consumer_group.go

```diff
@@ -543,6 +551,15 @@ func (s *consumerGroupSession) ResetOffset(topic string, partition int32, offset
 	}
 }
 
+func (s *consumerGroupSession) NextOffset(topic string, partition int32) (int64, string, error) {
+	if pom := s.offsets.findPOM(topic, partition); pom != nil {
```

Comment:
instead of retrieving this information, wouldn't make it more sense to capture the initial offsets at the beginning of the session, see line https://github.com/Shopify/sarama/pull/1271/files#diff-9076b00c0e67e4484d0c04fb247353c7L567

Reply:
@dim thanks for the review,
correct me if I'm wrong, this means we need to keep the state for each and every managed partition, which is already kept by the pom